### ISSUE TITLE
Fastboot support

### ISSUE THREAD

**Issue by benoror** [Glavin001/ember-c3#68]
When trying to use ember-c3 along Fastboot I get the following runtime error:

```
Error: Could not find module `d3` imported from `(require)`
    at missingModule (/Users/benoror/workspace/nimbox/tmp/broccoli_merge_trees-output_path-WSLosZ18.tmp/assets/vendor/loader/loader.js:219:1)
[...]
```

AFAIK the reason is that bower (requireJS) dependencies are deprecated. 

Maybe is just a matter of using a shim like [ember-d3](https://github.com/ivanvanderbyl/ember-d3) or [ember-c3-shim](https://github.com/mike-north/ember-c3-shim) ?

**Comment by maxwondercorn** [comment]
See issue #77

**Comment by maxwondercorn** [comment]
should be fixed with pr #88
